tests/keras/layers/wrappers_test.py

```diff
@@ -97,7 +97,7 @@ def test_TimeDistributed_learning_phase():
     y = wrappers.TimeDistributed(core.Dropout(.999))(x, training=True)
     model = Model(x, y)
     y = model.predict(np.random.random((10, 3, 2)))
-    assert_allclose(0., y, atol=1e-1, rtol=1e-1)
+    assert_allclose(y, np.zeros(y.shape), atol=1e-1, rtol=1e-1)
```

Comment:
I think you might have it backwards. [numpy docs for assert_allclose](https://docs.scipy.org/doc/numpy-dev/reference/generated/numpy.testing.assert_allclose.html) quoted here say: `numpy.testing.assert_allclose(actual, desired, rtol=1e-07, atol=0, equal_nan=True, err_msg='', verbose=True)`.

A better check would be with completely fixed random seeds for dropout so the same things are dropped every time. Then dropout could also be 0.5. I think TF can do that, can theano?